src/dictionaries.h

```diff
@@ -29,4 +29,20 @@ class DependentDictionary
         std::map<std::pair<std::string, std::string>, uint8_t> m_reverseDict;
 };
 
+class Dictionaries
+{
+    public:
+        Dictionaries();
+        const BasicDictionary& getAttributes() const;
```

Comment:
I think it would be better not to use `get` prefix in these methods. Usually, `get` implies `set`, but we do not have setters. Also, we are not getting anything here, we simpli refer some values. So `attributes` instead of `getAttributes`, `vendoerNames` instead of `getVendorNames`, and so on.

Also, since you simply return private members in these methods, you can define them inline:
```c++
const BasicDictionary& attributes() const { return m_attributes; }
```
No need to pollute cpp-file with one-liners.